platform-sdk/swirlds-config-impl/src/main/java/com/swirlds/config/impl/internal/ConfigurationImpl.java

```diff
@@ -121,6 +122,42 @@ public <T> List<T> getValues(final String propertyName, final Class<T> propertyT
         return getValues(propertyName, propertyType);
     }
 
+    @Override
+    public Set<String> getValueSet(final String propertyName) {
```

Comment:
Nullable/NonNull annotations on all methods

Reply:
Added to the interface

Reply:
added to impl

Reply:
https://github.com/hashgraph/hedera-services/pull/5814 does it to all methods that are not new for the given PR.